src/apps/chat/models.py

```diff
@@ -46,6 +65,9 @@ def get_upload_to(self, filename):
         return f"images/room/{self.id}/{filename}"
 
     user_attribute = "creator"
+    category = models.ManyToManyField(
```

Comment:
I think it's better to have `rooms = ManyToManyField` in `Category` model because `Room` shouldn't depend on `Category` (room can exist without category but not otherwise).